### PR TITLE
webapp: Adjust webapp addresses, configuration and docker images.

### PR DESCRIPTION
**Pull request by leckijakub**
After web services were packed inside docker container, configuration of addressed needed to be adjusted.
Also new images did not satisfied all dependencies of applications.

### CODE REVIEW

File: database/db.init [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+INSERT INTO SUBJECT (name, description, user_id)
+VALUES ('MBI-2022', 'Metody badawcze w informatyce', 1),
+       ('SOWW-2022', 'Systemy o wysokiej wydajności', 1),
+       ('BS-2022', 'Bezpieczeństwo systemów', 1),
+       ('BO-2022', 'Badania operacyjne', 1),
+       ('BS-2022', 'Biblioteki cyfrowe', 1),
+       ('UO-2022', 'Użyteczność oprogramowania', 1);
+
+INSERT INTO EXAM (name, description, subject_id, user_id, pass_score, date)
```
Comment (Debskij) [inline database/db.init:95]: Shouldn't we insert `owners` into `exam` to somehow link user to exam?

File: database/db.init [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+(
+    id          SERIAL PRIMARY KEY UNIQUE,
+    name        VARCHAR,
+    description VARCHAR,
+    subject_id  SERIAL REFERENCES SUBJECT (id),
+    pass_score  VARCHAR,
+    file_name   VARCHAR,
+    date        DATE,
+    user_id     SERIAL REFERENCES public."user" (id),
+    owners      VARCHAR
```
Comment (Debskij) [inline database/db.init:34]: How do we want to assign people who aren't the owner but are capable of modifying exams (e.g. by adding some students' exams)?

File: database/db.init [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    user_id SERIAL REFERENCES public."user" (id)
+);
+
+CREATE TABLE EXAM_FILES
+(
+    id            SERIAL PRIMARY KEY UNIQUE,
+    file_name     VARCHAR,
+    file_path     VARCHAR,
+    students_work bool,
+    answers       bool,
```
Comment (Debskij) [inline database/db.init:49]: Seems redundant. If it's not students' work, it has to be answers. What else could it be?

**Review by Debskij — CHANGES_REQUESTED** [review]
I am not capable of reviewing PHP. Dropped some comments on `db.init`.

File: database/db.init (hunk 1 above)
Comment (leckijakub) [reply database/db.init:95]: @MateuszKepczynski please have a look

File: database/db.init (hunk 2 above)
Comment (leckijakub) [reply database/db.init:34]: @MateuszKepczynski please have a look

File: database/db.init (hunk 3 above)
Comment (leckijakub) [reply database/db.init:49]: @MateuszKepczynski please have a look

**Review by marekhering — APPROVED** [review]

**Comment by leckijakub** [comment]
Merging as this version was presented to the supervisor and is a base for further changes.

**Pull request merged**